src/DataBoxEdge/DataBoxEdge/Common/Cmdlets/Share/DataBoxEdgeShareNewCmdletBase.cs

```diff
@@ -143,14 +186,33 @@ private PSResourceModel CreateResourceModel()
                 this.ResourceGroupName));
         }
 
+        private ResourceModel AddAzureContainer(ResourceModel resourceModel)
+        {
+            var sac = this.DataBoxEdgeManagementClient.StorageAccountCredentials.Get(
```

Comment:
please use meaningful name instead of sac

Reply:
Sure, will do